tests/test_openapi.py

```diff
@@ -0,0 +1,20 @@
+import json
+from pathlib import Path
+
+from generate_openapi import generate_openapi
+
+DIR_PROJECT_ROOT = Path(__file__).parent.parent
+DIR_TESTDATA = Path(__file__).parent / "testdata"
+
+
+def test_openapi_spec_current():
+    """Verify the openapi spec hasn´t changed."""
+
+    old_spec_path = DIR_PROJECT_ROOT / "openapi.json"
+    new_spec_path = DIR_TESTDATA / "openapi-new.json"
+    generate_openapi(new_spec_path)
```

Comment:
Could you use formatted json for openapi(-new).json?

I changed a comment in the source that ended up in the openapi-new.json. It is very hard to find the culprit in a one line diff that basically tells you the whole line changed.

Reply:
Sorry about that, spec is now pretty printed.